src/Controls/GlobeControls.js

```diff
@@ -762,23 +765,32 @@ class GlobeControls extends THREE.EventDispatcher {
     handleZoom(event) {
         this.player.stop();
         CameraUtils.stop(this.view, this.camera);
+        const zoomScale = event.delta > 0 ? this.zoomInScale : this.zoomOutScale;
+        let point = event.type === 'dolly' ? pickedPosition : this.view.getPickingPositionFromDepth(event.viewCoords);        // get cursor position
+        let range = this.camera.position.distanceTo(cameraTarget.position);
+        range *= zoomScale;
 
-        this.updateTarget();
-        const delta = -event.delta;
-        this.dolly(delta);
+        if (point && (range > this.minDistance && range < this.maxDistance)) {  // check if the zoom is in the allowed interval
```

Comment:
Perhaps this is caused by an issue in `range` calculation since when logging `range` in this situation, I get values that are way higher than `minDistance` (`680m` for a minimum distance of `250m`) and the camera still doesn't get closer to ground.

Reply:
Yes you are right, it seems there is issue at high zoom.
I've just did some change and it looks better.
I'm now using the `getRange` function.
It's been so long, I don't know why I didn't used it at first :thinking: but it looks better like this.